theano/gpuarray/__init__.py

```diff
@@ -123,7 +123,8 @@ def init_dev(dev, name=None, preallocate=None):
               (name, dev, context.devname, pcibusid),
               file=sys.stderr)
     pygpu_activated = True
-
+    cuda_available = True
```

Comment:
We don't use that variable anymore. This was the name in the old back-end. User should start to use pygpu_activated instead.

If this is difficult for user, can you explain why? To have this change work well, it would request other change to this file.